solidity/test/Consumer_test.js

```diff
@@ -1,6 +1,21 @@
-'use strict'
-
-require('./support/helpers.js')
+import cbor from 'cbor'
+import {
+  assertActionThrows,
+  consumer,
+  decodeRunRequest,
+  defaultAccount,
+  deploy,
+  eth,
+  functionSelector,
+  getLatestEvent,
+  hexToInt,
+  oracleNode,
+  rPad,
+  requestDataBytes,
+  requestDataFrom,
+  stranger,
+  toHex
+} from './support/helpers'
```

Comment:
This is quite a few functions. What would you say to `import * as helpers from `./support/helpers` and using a `helpers.` prefix in code?